scripts/check_endpoints/queries.py

```diff
@@ -0,0 +1,68 @@
+"""
+This file contains the required logic to
+call a simple query on a given endpoint
+based on the type of endpoint.
+"""
+
+from endpoints import Endpoint
+import requests
+from typing import Union
+
+
+def query_endpoint(ep: Endpoint) -> Union[bool, None]:
```

Comment:
What about websocket and Cosmos gRPC.

nit: not part of this PR, but it would be nice to rename Tendermint to Comet

Reply:
will create a follow-up issue to address the CometBFT names.